index.html

```diff
@@ -34,3 +34,223 @@ <h1 class="homepage-heading">Emmanuel <br>
             </div>
     
         </section>
+        <section class="projects">
```

Comment:
- [ ] The Project section is missing a closing tag `</section>` please double check it 👀 and kindly add It at the end of the project block.